test/info.py

```diff
@@ -140,6 +140,7 @@
     (   0x9011,     'lpc11u35_archmax_if',                      None,               'Seeed-Arch-Max'                        ),
     (   0x9012,     'lpc11u35_tiny_if',                         None,               'Seeed-Tiny-BLE'                        ),
     (   0x9013,     'lpc11u35_archlink_if',                     None,               'Seeed-Arch-Link'                       ),
+    (   0x9014,     'lpc11u35_96b_nitrogen_if',                 None,               'Seeed-96Boards-Nitrogen'               ),
```

Comment:
If you want this as part of the next DAPLink release you'll need to add it to the above PROJECT_RELEASE_INFO table as well.

Reply:
Added to the PROJECT_RELEASE_INFO table as suggested, thanks!